### DISCUSSION TITLE
howto change  the steeef theme for one line prompt?

### DISCUSSION THREAD

**Discussion by 2Belette** [zimfw/zimfw#109] [Q&A] [answered]
Description
-----------
This is more a question than a bug.
I really like zim in remplacement of my prezto, but I miss a little thing for the theme.
Steeef is almost the perfect theme, but I prefer to get everything on one line instead of getting the info the line before where I can actually type in.
I changed a little bit the PROMPT line on steeef.zsh-theme and I get exactly what I want but now I got a side effect when I use tab for autocompletion where the cursor is lost and delete from the screen a part of the command.
Is there a simple way to make it work correctly with my little change on steeef.zsh-theme ?
Many thanks and good job :)

Steps to reproduce
------------------
`  PROMPT='%{$purple%}%n${${reset_color}%} at %{$orange%}%m${${reset_color}%} in %{$limegreen%}%~${${reset_color}%} $vcs_info_msg_0_$(virtualenv_info)%{${reset_color}%}%(!.#.$) '`

![tab](https://cloud.githubusercontent.com/assets/2325706/21027190/04103f58-bd90-11e6-9e72-292432cc9f6a.png)

**Comment by Plloi** [comment, marked as answer]
Looks like some of the color resets are not marked properly try the following:
`PROMPT='%{$purple%}%n%{${reset_color}%} at %{$orange%}%m%{${reset_color}%} in %{$limegreen%}%~%{${reset_color}%} $vcs_info_msg_0_$(virtualenv_info)%{${reset_color}%}%(!.#.$) '`

**Comment by 2Belette** [comment]
I think you make my day :) :)
many thanks

**Comment by Eriner** [comment]
I believe @Plloi has answered your question, so I'm going to go ahead and close this issue. Thanks for the assistance, @Plloi!
